vasyl_varyshchuk/Task_2/pass_arg_decorator.py

```diff
@@ -0,0 +1,39 @@
+"""
+This module implements a retry decorator with ability to pass a parameter into decorator
+"""
+
+import random
+
+
+def decorator(retries=None):
+
+    def my_decorator(func):
+
+        def func_wrapper():
+            if retries is not None:
+                for i in range(1, retries+1):
```

Comment:
What is the reason of range `(1, retries +1)` instead of the default `(0, retries)`?

Reply:
I used it only to print the number of retries, but it is better to use  range(0, retries) and then use
print('This text prints when exception is not raised, i = {}'.format(i+1))
instead of 
print('This text prints when exception is not raised, i = {}'.format(i))

Reply:
Exactly! It's much better to increase where you really need it instead of adding some complexity into base code.